src/api/app/controllers/webui/kiwi/images_controller.rb

```diff
@@ -43,16 +43,20 @@ def update
         end
         redirect_to action: :show
       rescue ActiveRecord::RecordInvalid, Timeout::Error => e
-        flash[:error] = "Cannot update repositories for kiwi image: #{@image.errors.full_messages.to_sentence} #{e.message}"
+        flash[:error] = "Cannot update kiwi image: #{@image.errors.full_messages.to_sentence} #{e.message}"
         redirect_back(fallback_location: root_path)
       end
 
       private
 
       def image_params
         params.require(:kiwi_image).permit(repositories_attributes:
```

Comment:
`Layout/MultilineArrayBraceLayout`, I just open an issue in Rubocop.

That's why we wrote it in that strange way, but with the code you suggested it works, so we do not need to disable in out project :+1: